core/server/common/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -91,15 +97,17 @@
         MASTER_WEB_VALIDATION_TASK,
         PROXY_WEB_VALIDATION_TASK,
         MASTERS_SSH_VALIDATION_TASK,
-        WORKERS_SSH_VALIDATION_TASK
+        WORKERS_SSH_VALIDATION_TASK,
```

Comment:
The rearrangement of the task lists is in PR https://github.com/Alluxio/alluxio/pull/5865